contentcuration/contentcuration/models.py

```diff
@@ -2674,12 +2674,6 @@ def filter_edit_queryset(cls, queryset, user):
         return queryset.filter(author=user, channel__editors=user)
 
     class Meta:
-        constraints = [
-            models.UniqueConstraint(
-                fields=["channel", "channel_version"],
-                name="unique_channel_with_channel_version",
-            ),
-        ]
```

Comment:
Whoops, it seems I didn't explain this well 😅. Apologies. This constraint still applies; we should not allow creating multiple submissions of the same channel version. If the user wants to create a new submission for the same channel, they must publish a new version first.